### PR TITLE
[DEV APPROVED] Removed baby cost calc

### PR DESCRIPTION
**Pull request by habitullence**

### CODE REVIEW

**Comment by Tr4pSt3R** [comment]
:+1:

File: .node-version [hunk 1]
```diff
@@ -1 +1 @@
-0.10.26
+0.10.36
```
Comment (tomas-stefano) [inline .node-version]: It should be included on this PR?

**Comment by tomas-stefano** [comment]
:+1:

File: app/controllers/baby_cost_calculator_controller.rb [hunk 2]
```diff
@@ -1,4 +1,10 @@
 class BabyCostCalculatorController < EmbeddedToolsController
+  before_action :deny_non_syndicates
+
+  def deny_non_syndicates
+    redirect_to main_app.article_path('baby-costs-calculator'), status: 301 if syndicated_tool_request? == false
```
Comment (tomas-stefano) [inline app/controllers/baby_cost_calculator_controller.rb]: I would use `unless syndicated_tool_request?` and then you dont need the check `== false`

Comment (jongilbraith) [inline app/controllers/baby_cost_calculator_controller.rb]: Or ``if !syndicated_tool_request?``, but either way would be good to do away with the explicit `==`.

File: spec/requests/tool_integration/baby_cost_calculator_spec.rb [hunk 3]
```diff
@@ -1,14 +1,29 @@
...
-        get path
+    en
+    cy
+  ).each do |locale|
+    context "locale #{locale}" do
+      let (:path) { "/#{locale}/tools/" + "ToolMountPoint::BabyCostCalculator::#{locale.upcase}_ID".constantize }
+
+      describe 'non-syndicated tool request' do
+        before do
+          get path, nil, { 'X-Syndicated-Tool' => false }
```
Comment (jongilbraith) [inline spec/requests/tool_integration/baby_cost_calculator_spec.rb]: Do we still get the header at all on non syndicated requests? If not, might be appropriate to make the test mimic that.

File: spec/requests/tool_integration/baby_cost_calculator_spec.rb [hunk 4]
```diff
@@ -1,14 +1,29 @@
...
+        end
+
+        it "redirects to article baby cost calculator" do
+          expect(response.status).to eq 301
+          expect(response).to redirect_to article_path('baby-costs-calculator', locale: locale)
```
Comment (Tr4pSt3R) [inline spec/requests/tool_integration/baby_cost_calculator_spec.rb]: Hey @habitullence this redirect will work but, with a hidden gotcha :)

Whereas, https://www.moneyadviceservice.org.uk/en/articles/baby-costs-calculator will work. The Welsh link https://www.moneyadviceservice.org.uk/cy/articles/baby-costs-calculator will not work. 

The correct welsh link is https://www.moneyadviceservice.org.uk/cy/articles/cyfrifiannell-costau-babi

Comment (habitullence) [inline spec/requests/tool_integration/baby_cost_calculator_spec.rb]: Good catch, I've updated the PR.

**Comment by Tr4pSt3R** [comment]
Just @ me when it's pushed and I will thumb it

File: spec/requests/tool_integration/baby_cost_calculator_spec.rb (hunk 3 above)
Comment (habitullence) [inline spec/requests/tool_integration/baby_cost_calculator_spec.rb]: You're right, header should either be there or not.

**Comment by habitullence** [comment]
How do you organise your localisation files?

**Comment by Tr4pSt3R** [comment]
@habitullence there are no fast rules.
 
Have a look at this simple example.
https://github.com/moneyadviceservice/frontend/blob/master/config/locales/en.yml#L371

**Comment by habitullence** [comment]
Updated with I18n.

**Comment by tomas-stefano** [comment]
:+1: :+1:

**Comment by Tr4pSt3R** [comment]
:+1: :+1: :+1:

**Pull request merged**